src/Symfony/Component/Security/Core/Authentication/Token/AbstractToken.php

```diff
@@ -267,9 +258,9 @@ final public function serialize(): string
     /**
      * @internal
      */
-    final public function unserialize($serialized)
+    final public function unserialize(string $serialized)
     {
-        $this->__unserialize(\is_array($serialized) ? $serialized : unserialize($serialized));
```

Comment:
Are we sure we can safely remove support for arrays here?

Reply:
yes, this is legacy from v4.4